tests/select.spec.ts

```diff
@@ -206,7 +206,7 @@ test('When there is a reduce sort option, the elements should reduce on matching
   );
 });
 
-test('On pressing enter over an option, that option should be selected', async ({
+test('On pressing enter over an option, that option should be selected, change event emitted', async ({
```

Comment:
Looks like this was already the case, but I don't think this title is correct, test appears to be be pressing an arrow, not [enter]`.

If a down arrow is pressed, I think the spec should be only that an `input` event is emitted (because the value changed), but not a `change` event, because the user has not selected the option yet by pressing enter or clicking away. What do you think?

Reply:
on the title and existing test, it already has the portion where the user presses down the 'enter' key, where they select the option and the value is changed, shouldn't a `change` event and an `input` event be expected?

Reply:
My bad, missed the line where enter was pressed